FortnoxAPILibrary.Tests/Connectors/ConnectorTestsBase.cs

```diff
@@ -5,8 +5,8 @@ namespace FortnoxAPILibrary.Tests.Connectors
     public abstract class ConnectorTestsBase
     {
         //Enter these values to run tests. DO NOT CHECK THESE CREDENTIALS IN. Also make sure this is not production account, these tests will make changes to the data.
-        protected static string AccessToken = "";
-        protected static string ClientSecret = "";
+        protected static string AccessToken = "9a4d0895-9877-4ef4-9a90-04787e8e17ad";
+        protected static string ClientSecret = "PNWPB5EGzJ";
```

Comment:
As I know, it is automatically applied for build in Teamcity and comment above:
//Enter these values to run tests. DO NOT CHECK THESE CREDENTIALS IN